driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -151,6 +152,14 @@
      */
     private ExecutorService threadPool;
 
+    /*
+     * Version strings for connected proxy and store
+     * Used mainly for tests
+     */
+    private String proxyVersion;
```

Comment:
These are not thread safe as-is. They should arguably be volatile at least but using them at all very much implies that the test environment is unchanging and that the result will be the same on every request. Something different will need to be done if this information is ever surfaced via public API